### PR TITLE
Translate sponsors footer in french

### PR DESCRIPTION
**Pull request by benoittgt**
It was weird to have mixing languages on landing page.

Screenshot 2022-06-23 at 07 50 59

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -8,10 +8,15 @@ fr:
     Ruby. Publiez vos gems instantanément et utilisez-les. Utilisez l'API pour interagir
     et trouver des informations sur les gems disponibles. Contribuez et améliorez
     ce site avec nous !
-  footer_sponsors_html:
-  footer_join_rt_html:
+  footer_sponsors_html: RubyGems.org est rendu possible grâce à un partenariat avec une grande communauté Ruby.
```
Comment (jessicajoly88) [inline config/locales/fr.yml:11]: avec "la communauté" Ruby sounds a bit better but nitpick.

File: config/locales/fr.yml [hunk 2]
```diff
@@ -8,10 +8,15 @@ fr:
...
+    <a href="https://www.fastly.com/">Fastly</a> fournisseur de bande passante et du support CDN,
+    <a href="https://www.rubycentral.org/">Ruby Central</a> qui couvre les frais d'infrastructure, et
+    <a href="https://rubytogether.org/?source=rubygems">Ruby Together</a> qui finance le développement et le travail d'ops.
```
Comment (jessicajoly88) [inline config/locales/fr.yml:14]: `le travail d'ops`  is there a more formal term for this? I'm guessing ops is operations but it's still not clear to me.

File: config/locales/fr.yml [hunk 3]
```diff
@@ -8,10 +8,15 @@ fr:
...
+    <a href="%{sponsors_page}">Apprenez en plus sur nos sponsors et comment ils travaillent entre eux.</a>.
```
Comment (jessicajoly88) [inline config/locales/fr.yml:15]: `commanditaires` au lieu de sponsors (english term).

**Review by jessicajoly88 — COMMENTED** [review]
Thanks for taking this on 💯. I left some comments.

File: config/locales/fr.yml (hunk 3 above)
Comment (casperisfine) [reply config/locales/fr.yml:15]: It's commonly used in `FR_fr` though. Most french people wouldn't understand `commanditaires`.

File: config/locales/fr.yml (hunk 2 above)
Comment (casperisfine) [reply config/locales/fr.yml:14]: I'd suggest: `et l'administration des serveurs.`

**Comment by benoittgt** [comment]
Thanks for your feedbacks @jessicajoly88 and @casperisfine.

File: config/locales/fr.yml [hunk 4]
```diff
@@ -8,10 +8,15 @@ fr:
     Ruby. Publiez vos gems instantanément et utilisez-les. Utilisez l'API pour interagir
     et trouver des informations sur les gems disponibles. Contribuez et améliorez
     ce site avec nous !
-  footer_sponsors_html:
-  footer_join_rt_html:
+  footer_sponsors_html: RubyGems.org est rendu possible grâce à un partenariat la communauté Ruby.
```
Comment (etiennebarrie) [inline config/locales/fr.yml:11]:
```suggestion
  footer_sponsors_html: RubyGems.org est rendu possible grâce à un partenariat avec la communauté Ruby.
```
If you want to keep the "greater", you can use "au sens large" or "élargie"

Comment (benoittgt) [reply config/locales/fr.yml]: Good idea. [`2ccd27b` (#3112)](https://github.com/rubygems/rubygems.org/pull/3112/commits/2ccd27be336278d29fd44514466c7e1c0db9703b)

File: config/locales/fr.yml [hunk 5]
```diff
@@ -8,10 +8,15 @@ fr:
     Ruby. Publiez vos gems instantanément et utilisez-les. Utilisez l'API pour interagir
     et trouver des informations sur les gems disponibles. Contribuez et améliorez
     ce site avec nous !
-  footer_sponsors_html:
-  footer_join_rt_html:
+  footer_sponsors_html: RubyGems.org est rendu possible grâce à un partenariat avec la communauté Ruby au sens large.
+    <a href="https://www.fastly.com/">Fastly</a> fournisseur de bande passante et du support CDN,
+    <a href="https://www.rubycentral.org/">Ruby Central</a> qui couvre les frais d'infrastructure, et
+    <a href="https://rubytogether.org/?source=rubygems">Ruby Together</a> qui finance le développement et l'administration des serveurs.
+    <a href="%{sponsors_page}">Apprenez en plus sur nos sponsors et comment ils travaillent entre eux.</a>.
```
Comment (jenshenny) [inline config/locales/fr.yml:15]:
```suggestion
    <a href="%{sponsors_page}">Apprenez en plus sur nos sponsors et comment ils travaillent entre eux</a>.
```
Extra `.`
Screen Shot 2022-07-05 at 2 43 11 PM

Comment (benoittgt) [reply config/locales/fr.yml]: Good catch. Thanks @jenshenny

**Comment by benoittgt** [comment]
I'm happy to know more people from Shopify 😄💕

**Comment by jenshenny** [comment]
Thanks @benoittgt for opening this! @jessicajoly88 @etiennebarrie @casperisfine could one of you go through this one last time and give a 👍 if it looks good?

**Comment by byroot** [comment]
👍

**Pull request merged**